src/redux/actionCreators/assignAction.js

```diff
@@ -0,0 +1,119 @@
+import axios from 'axios';
+import { toast } from 'react-toastify';
+import i18next from 'i18next';
+import {
+    GET_DRIVERS_FAILED,
+    GET_DRIVERS_SUCCESS,
+  
+    
+    ASSIGN_BUSDRIVER_FAILED,
+    ASSIGN_BUSDRIVER_SUCCESS,
+    
+    UNASSIGN_BUSDRIVER_FAILED,
+  
+    UNASSIGN_BUSDRIVER_SUCCESS,
+
+    GET_BUS_FAILED,
+    GET_BUS_SUCCESS,
+    
+} from '../actionTypes/actionTypes';
+import {token} from '../../components/AssignBusToDriver/token';
+
+export const getDriversAction = () => async(dispatch) => {
+    
+    
+    try {
+        const response = await axios
+        .get('https://incredible-phantom.herokuapp.com/api/users',{
```

Comment:
we stored this url in a constant called `BACKEND_URL` you can import it and use it